src/modules/numbers/numbers.service.ts

```diff
@@ -35,7 +37,7 @@ export class NumbersService {
       if (!dbNumber[0]) {
```

Comment:
Can this please be replaced with `dbNumber && !dbNumber[0]`, in case `dbNumber` is not an array.